code/modules/economy/Accounts_DB.dm

```diff
@@ -254,8 +251,7 @@
 							<tbody>
 					"}
 
-					for(var/i=1, i<=SSeconomy.all_money_accounts.len, i++)
-						var/datum/money_account/D = SSeconomy.all_money_accounts[i]
+					for(var/datum/money_account/D in SSeconomy.all_money_accounts)
```

Comment:
This won't work. Because `all_money_accounts` is a list of `string => datum`, the for-each loop iterates over the _keys_. So to get the datum, at minimum, you'd need `for(var/M in SSeconomy.all_money_accounts) { var/datum/money_account/D = SSeconomy.all_money_accounts[M]`

Reply:
Yup, missed that.